packages/namekit-react/src/components/DomainCardPrice.tsx

```diff
@@ -0,0 +1,41 @@
+import { DisplayedPrice, DisplayedPriceProps } from "./DisplayedPrice";
+import {
+  CurrencySymbology,
+  getCurrencySymbology,
+  Price,
+} from "@namehash/ens-utils";
+import { Tooltip } from "./Tooltip";
+import React from "react";
+
+interface DomainCardPriceProps extends DisplayedPriceProps {
+  altPrice: Price;
+  altPriceSymbology: CurrencySymbology;
+}
+
+export const DomainCardPrice = ({
```

Comment:
I like what you've done with this UI component :+1:

We shouldn't call it a `DomainCardPrice` though because this idea isn't specific to `DomainCard`.

Maybe `DisplayedPriceConversion`? Assuming so we should also rename `altPrice` and related fields to something such as `convertedPrice`, etc..

Reply:
Sounds good, @lightwalker-eth, 🔝 I will do so